diff -p -up xf86-video-openchrome-0.2.903/src/via_id.c.na01 xf86-video-openchrome-0.2.903/src/via_id.c
--- xf86-video-openchrome-0.2.903/src/via_id.c.na01	2009-09-28 08:45:52.000000000 +0200
+++ xf86-video-openchrome-0.2.903/src/via_id.c	2009-09-28 08:46:40.000000000 +0200
@@ -225,6 +225,7 @@ static struct ViaCardIdStruct ViaCardId[
     {"VIA OpenBook",                          VIA_VX800,   0x1170, 0x0311, VIA_DEVICE_CRT | VIA_DEVICE_LCD}, /* VIA OpenBook eNote VBE8910 */
     {"Samsung NC20",                          VIA_VX800,   0x144d, 0xc04e, VIA_DEVICE_CRT | VIA_DEVICE_LCD},
     {"Quanta DreamBook Light IL1",            VIA_VX800,   0x152d, 0x0771, VIA_DEVICE_CRT | VIA_DEVICE_LCD},
+    {"Guillemot NA01",                        VIA_VX800,   0x1106, 0x3349, VIA_DEVICE_CRT | VIA_DEVICE_LCD},
     {"Lenovo S12",                            VIA_VX800,   0x17aa, 0x388c, VIA_DEVICE_CRT | VIA_DEVICE_LCD},
 
     /*** VX855 ***/
