--- xf86-video-openchrome-0.3.1/configure.ac~	2013-02-22 17:55:23.605298532 +0100
+++ xf86-video-openchrome-0.3.1/configure.ac	2013-02-22 17:56:01.423772752 +0100
@@ -38,7 +38,7 @@ AC_DEFINE_UNQUOTED([VIA_PATCHLEVEL],
                    [Patch version])
 
 AC_CONFIG_SRCDIR([Makefile.am])
-AM_CONFIG_HEADER([config.h])
+AC_CONFIG_HEADER([config.h])
 AC_CONFIG_AUX_DIR(.)
 
 AM_INIT_AUTOMAKE([foreign dist-bzip2])
